client/src/common/types/Course.ts

```diff
@@ -3,6 +3,11 @@
  */
 interface Course {
     name: string;
+    courseID: string;
+    description: string;
+    credithours: number;
+    prereq:  string[];
```

Comment:
The data represents it as a string, should we make it a string array?

Reply:
What I mean by that, is that should we take it in as a string, and then when we need to utilize it, we split it into an array.

Reply:
Sure we can do that